test/cmdlineTests.sh

```diff
@@ -133,37 +159,44 @@ function test_solc_behaviour()
     if [[ $exitCode -ne "$exit_code_expected" ]]
     then
         printError "Incorrect exit code. Expected $exit_code_expected but got $exitCode."
-        rm -f $stdout_path $stderr_path
         exit 1
     fi
 
     if [[ "$(cat $stdout_path)" != "${stdout_expected}" ]]
```

Comment:
Could this block and the one starting in line 183 generalized into a function?

Reply:
Yes, but maybe we can do it later when we also want auto-correction for yul command line test, etc?

Reply:
Sounds good!